include/ringmesh/basic/frame.h

```diff
@@ -152,6 +152,80 @@ namespace RINGMesh
     };
     ALIAS_2D_AND_3D( ReferenceFrame );
 
+    template < index_t DIMENSION >
+    class RINGMESH_API FrameManipulator
```

Comment:
This manipulator does not work with Frame but ReferenceFrame. So why not ReferenceFrameManipulator?

Reply:
true